book.rb

```diff
@@ -0,0 +1,16 @@
+require 'date'
+require './rental'
```

Comment:
- Kindly In order to include classes, modules, etc in other files you have to use require_relative or require, and when it is some dependency on locating in gem kindly use require `eg: require 'date'.` so here above  I suggest you use require_relative instead of required  for rental `eg: require_relative  './rental'`  and for  date file use require  ex: require 'date' it will fix the error you are facing when running your main.rb file  :+1: :+1: :+1: